src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -381,4 +394,20 @@ public static void deleteScratchFile(FlightContext context, GcsPdao gcsPdao) {
     String pathToIngestFile = workingMap.get(IngestMapKeys.INGEST_CONTROL_FILE_PATH, String.class);
     gcsPdao.deleteFileByGspath(pathToIngestFile, bucketResource.projectIdForBucket());
   }
+
+  public static void validateBulkLoadFileModel(BulkLoadFileModel loadFile) {
+    List<String> itemsNotDefined = new ArrayList<>();
+    if (loadFile.getSourcePath() == null || loadFile.getSourcePath().isEmpty()) {
```

Comment:
I think you can just use `StringUtils.isEmpty(loadFile.getSourcePath())` to do both of these (note: same for targetPath below)

Reply:
These fields are annotated as `@NotNull`, are the null checks necessary?

Reply:
And thanks @nmalfroy  - i was looking for that in the regular string methods and forgot it was using StringUtils. Switching now.